README.md

```diff
@@ -27,6 +27,10 @@ cd deepcell-label
 
 The backend is a Flask HTTP API that will store and update project data.
 
+#### Set up environment
+
+Flask expects some environment variables to be set, like FLASK_APP. Set these with `export FLASK_APP=application` in your terminal, or set them in the `.env` file in the root of the repository. There is an example `.env.example` with FLASK_APP already set. Make a copy as remove `.example` so the file is named `.env`.
```

Comment:
```suggestion
Flask expects some environment variables to be set, like FLASK_APP. Set these with `export FLASK_APP=application` in your terminal, or set them in the `.env` file in the root of the repository. There is an example `.env.example` with FLASK_APP already set. Make a copy and remove `.example` so the file is named `.env`.
```

Reply:
also, consider adding a brief explanation of why `application` is the correct value. (Or link to the flask docs that talks about the FLASK_APP var)